packages/firebase_data_connect/firebase_data_connect/test/src/common/common_library_test.dart

```diff
@@ -28,14 +29,14 @@ void main() {
       const packageVersion = '1.0.0';
       expect(
         getGoogApiVal(CallerSDKType.core, packageVersion),
-        'gl-dart/$packageVersion fire/$packageVersion',
+        'gl-dart/$packageVersion fire/$packageVersion gl-${Platform.operatingSystem}',
       );
     });
     test('should return codegen suffix if using gen sdk', () {
       const packageVersion = '1.0.0';
       expect(
         getGoogApiVal(CallerSDKType.generated, packageVersion),
-        'gl-dart/$packageVersion fire/$packageVersion dart/gen',
+        'gl-dart/$packageVersion fire/$packageVersion dart/gen gl-${Platform.operatingSystem}',
```

Comment:
Assuming we aren't testing for 'web' here .

Reply:
Yeah, there's no way to run the test in `web` mode as of right now, and I don't think it's worth mocking and testing for.